### PR TITLE
Add sample: Translate text with premium model

### PR DESCRIPTION
**Pull request by remi**
https://cloud.google.com/translate/docs/premium#sample_code

### CODE REVIEW

**Comment by remi** [comment]
@frankyn PTAL

File: translate/translate_samples.rb [hunk 1]
```diff
@@ -28,6 +28,22 @@ def translate_text project_id:, text:, language_code:
   # [END translate_text]
 end
 
+def translate_text_with_model project_id:, text:, language_code:
+  # [START translate_text_with_model]
+  # project_id   = "Your Google Cloud project ID"
```
Comment (remi) [inline translate/translate_samples.rb]: Oh noes!  remi, yo... fix this!

File: translate/translate_samples.rb [hunk 2]
```diff
@@ -28,6 +28,22 @@ def translate_text project_id:, text:, language_code:
...
+  translation = translate.translate text, to: language_code, model: "nmt"
+
+  puts "Translated '#{text}' to '#{translation.text.inspect}'"
+  puts "Original language: #{translation.from} translated to: #{translation.to}"
+  # [END translate_text_with_model]
+end
+
 def detect_language project_id:, text:
   # [START detect_language]
   # project_id   = "Your Google Cloud project ID"
```
Comment (frankyn) [inline translate/translate_samples.rb]: Alignment here

**Comment by remi** [comment]
Fixed & fixed a few other alignments that were out of whack in the file ~ PTAL 😄

**Comment by frankyn** [comment]
LGTM!

**Pull request merged**